netplugin/plugin/netplugin.go

```diff
@@ -75,8 +74,10 @@ func (p *NetPlugin) Init(pluginConfig Config) error {
 
 	// set state driver in instance info
 	pluginConfig.Instance.StateDriver = p.StateDriver
-
-	InitGlobalSettings(p.StateDriver, &pluginConfig.Instance)
+	err = InitGlobalSettings(p.StateDriver, &pluginConfig.Instance)
+	if err != nil {
+		return err
```

Comment:
Log error?

Reply:
Also if global setting != instant setting, are we going to do anything?

Reply:
it will be handled here 
```
	// Init the driver plugins..
	err = netPlugin.Init(*pluginConfig)
	if err != nil {
		log.Fatalf("Failed to initialize the plugin. Error: %s", err)
	}
```
which will exit(1) after log fatal